draft-scenarios/coops/the_ambushed_accord.tex

```diff
@@ -43,40 +43,44 @@ \subsection*{\MakeUppercase{Victory Conditions}}
 \subsection*{\MakeUppercase{Timed Events}}
 \textbf{\nth{2} Round:}
 \begin{itemize}
-  \item Build either \svg{building_special_tent} or \svg{bulding_special_gears} for free.
+  \item Build either Camp (\svg{building_special_tent}) or Workshop (gears) for free. %(\svg{bulding_special_gears})
 \end{itemize}
-\textbf{\nth{4} Round:}
+End of \textbf{\nth{4} Round:}
 \begin{itemize}
-  \item \textbf{One} player must fight a Timed Event Fight (TEF) of 5 × \svg{bronze}.
+  \item \textbf{one player} must fight a \textbf{Timed Event Fight} of 5 × \svg{bronze}.
 \end{itemize}
 End of \textbf{\nth{8} Round:}
 \begin{itemize}
-  \item \textbf{One} player must fight a TEF of 4 × \svg{silver}
+  \item \textbf{one player} must fight a \textbf{Timed Event Fight} of 4 × \svg{silver}
 \end{itemize}
 End of \textbf{\nth{10} Round:}
 \begin{itemize}
-  \item \textbf{One} player must fight a TEF of 3 × \svg{golden}.
+  \item \textbf{one player} must fight a \textbf{Timed Event Fight} of 3 × \svg{golden}.
 \end{itemize}
 End of \textbf{\nth{12} Round:}
 \begin{itemize}
-  \item \textbf{One} player must fight a TEF of 2 × \svg{azure}.
+  \item \textbf{one player} must fight a \textbf{Timed Event Fight} of 2 × \svg{azure}.
 \end{itemize}
 \clearpage
 
 \subsection*{\MakeUppercase{Additional Rules}}
-\subsubsection*{\MakeUppercase{Difficulty Settings}}
-\textbf{Easy:}
-\begin{itemize}
-  \item Remove a Unit from each Timed Event Fight.
-\end{itemize}
-\textbf{Hard:}
-\begin{itemize}
-  \item Add a initiative stack token to a random Unit of the TEFs on Round 4, 8 and 10.
-\end{itemize}
-\textbf{Impossible:}
-\begin{itemize}
-  \item Add two random stack tokens to two Units in each TEF fights.
-\end{itemize}
+\hommtablemulticol[]{21}{
+	\centering
+	\medskip
+	\textbf{Difficutly Settings}\\
+	\bigskip
+	
+	\begin{tabularx}{0.93\linewidth}{p{0.28\linewidth}X}
+		\darkcell[1.6]{Difficulty} & \darkcell[1.6]{Changes to Timed Event Fights} \\
+		\darkcell[1.6]{Easy}
+		& \lightcell[1.6]{Remove a random unit from each fight.} \\
+		\darkcell[2.2]{Hard}
```

Comment:
I wouldn't skip normal, as its absence is confusing. It's better to have an explicit "No changes" row.